### PR TITLE
🖥 Add Support for Desktop

### PR DESCRIPTION
**Pull request by YazeedAlKhalaf**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -15,6 +15,13 @@ dependencies:
   crypto: ^2.1.3
   pedantic: ^1.8.0
 
+  # Path Provider For Desktop (Needed when building for macOS, Windows, Linux)
+  path_provider_fde:
+    git:
+      url: git://github.com/google/flutter-desktop-embedding.git
+      path: plugins/flutter_plugins/path_provider_fde
+      ref: 282bb7b172a42a1f4b573540cc19e7610a867a93
```
Comment (stuartmorgan) [inline pubspec.yaml:23]: A package can't be published with a git reference.

Comment (YazeedAlKhalaf) [reply pubspec.yaml:23]: Yea I see your point. Never thought about it but yea 😎
Just a piece of advice; add this to the readme at least so people know that Windows is supported but in a non-official way.
Just a thought.
Thanks 😁

Comment (stuartmorgan) [reply pubspec.yaml:23]: That README already exists:
https://github.com/google/flutter-desktop-embedding/blob/master/plugins/flutter_plugins/README.md
That's referenced from the [official instructions for Flutter for Windows](https://github.com/flutter/flutter/wiki/Desktop-shells#plugins).

There are currently [234 plugins that depend on path_provider](https://pub.dev/packages?q=dependency%3Apath_provider). Updating all of their READMEs to mention path_provider_fde is impractical.

Comment (YazeedAlKhalaf) [reply pubspec.yaml:23]: Ah okay

**Pull request closed without merging**